### PR TITLE
Created Algorithm for randomised feed

### PR DESCRIPTION
**Pull request by Tom-Tee**
Hi guys,

Was a tough one this. The code is a little messy (I plan on making it look beautiful eventually 💋 ) but we now have a randomized feed on every refresh! Not perfect yet but I think it'll do for our mid-way demo.

Was going to get stuck into the logic of the interest between users but haven't got that far yet. I Will attempt very soon.

I've also made some components folders that aren't being used at the moment. Leave me with them!

Cheers,

Tom

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -9,16 +9,17 @@ def about
 
   def feed
     @user = current_user
-    @users = User.all
-    @events = Event.all
+    @users = User.all.sample(10)
+    @user = User.all
```
Comment (julianharr) [inline app/controllers/pages_controller.rb:13]: What is @user = User.all giving access to?

File: app/views/pages/feed.html.erb [hunk 2]
```diff
@@ -7,60 +9,102 @@
...
+    <% charity_array = [] %>
+    <% event_array = [] %>
+    <% @charity.each do |charity| %>
+    <% charity_array << charity %>
+    <% end %>
+    <% @events.each do |event| %>
+    <% event_array << event %>
+    <% end %>
+    <% array.take(10) %>
+
```
Comment (julianharr) [inline app/views/pages/feed.html.erb:39]: This is dope. Clever work.

**Comment by julianharr** [comment]
Looks great @Tom-Tee love your work.

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (matoni109) [reply app/controllers/pages_controller.rb:13]: maybe needs to be find_by id ?

**Pull request merged**